src/generate/apiCollection.test.ts

```diff
@@ -23,19 +23,17 @@ describe("Test ApiGroup class init", () => {
     const collection = await ApiCollection.init({
```

Comment:
chai-as-promised is no longer used in this file and can be removed.

Reply:
It's still used for the rejects tests at the bottom